### PR TITLE
Feature/populate dialog window

### PR DESCRIPTION
**Pull request by SNujiy**
Vis innhold i dialogvindu.

### CODE REVIEW

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 1]
```diff
@@ -1,32 +1,148 @@
...
+};
+
+const tabLabelItems = (
+  tabContents: DialogContent,
+  translation: Translation,
+): JSX.Element[] => {
+  const items = [];
+  tabContents.text
+    ? items.push(
+        <Trigger value="Text" className={styles.trigger}>
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:47]: Should these values be translated?

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 2]
```diff
@@ -1,32 +1,148 @@
...
+      <List className={styles.list} aria-label="tabs list">
+        {tabLabelItems(tabContents, translation)}
       </List>
-      {tabContents.map((el, i) => (
-        <Content key={i} value={el.title}>
-          {" "}
-          {el.content}{" "}
-        </Content>
-      ))}
+      <div style={{ overflowY: "scroll", overflow: "auto", height: "100%" }}>
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:143]: Move static styles into scss file

**Comment by boyum** [comment]
I think we should remove the description beneath the title, and only show it within the tabs.

image

**Comment by SNujiy** [comment]
> I think we should remove the description beneath the title, and only show it within the tabs.
> 
> image

This behavior is actually caused by some leftover codes. the description should only be visible above the tabs. 

Do you mean that we should add a "Description" tab and move it as part of the tabs?

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx (hunk 1 above)
Comment (SNujiy) [reply src/components/Dialog-Window/Tabs/DialogTabs.tsx:47]: I don't think translation are needed here since value and keys are not visible for end user?

**Comment by boyum** [comment]
> > I think we should remove the description beneath the title, and only show it within the tabs.
> > image
> 
> This behavior is actually caused by some leftover codes. the description should only be visible above the tabs.
> 
> Do you mean that we should add a "Description" tab and move it as part of the tabs?

I think it's designed to have the description within the `text` tab. Maybe @henriettemoe can shine some light on the situation.
image

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx (hunk 1 above)
Comment (boyum) [reply src/components/Dialog-Window/Tabs/DialogTabs.tsx:47]: Oooh, I see.

**Comment by SNujiy** [comment]
> > > I think we should remove the description beneath the title, and only show it within the tabs.
> > > image
> > 
> > 
> > This behavior is actually caused by some leftover codes. the description should only be visible above the tabs.
> > Do you mean that we should add a "Description" tab and move it as part of the tabs?
> 
> I think it's designed to have the description within the `text` tab. Maybe @henriettemoe can shine some light on the situation. image

Agreed on today's stand up that we will be following the original design and move description as part of "tabs" under Text

**Review by boyum — APPROVED** [review]

**Pull request merged**